docs/_includes/nav_footer_custom.html

```diff
@@ -1,8 +1,8 @@
 <footer class="site-footer">
   <a href="https://github.com/smart-on-fhir/cumulus">
   <picture>
-    <source srcset="assets/github-mark-white.svg" media="(prefers-color-scheme: dark)" />
-    <img style="height: 1em; width: auto" src="assets/github-mark.svg" alt="" />
+    <source srcset="/cumulus/assets/github-mark-white.svg" media="(prefers-color-scheme: dark)" />
+    <img style="height: 1em; width: auto" src="/cumulus/assets/github-mark.svg" alt="" />
```

Comment:
Unrelated, but I noticed that these images weren't being picked up in subfolders